lib/OpenLayers/Control/PanZoom.js

```diff
@@ -82,7 +82,9 @@ OpenLayers.Control.PanZoom = OpenLayers.Class(OpenLayers.Control, {
      */
     setMap: function(map) {
         OpenLayers.Control.prototype.setMap.apply(this, arguments);
-        this.map.events.register("buttonclick", this, this.onButtonClick);
+        
+        this.events.attachToElement(this.outsideViewport ? this.div : this.map.div);
+        this.events.register('buttonclick', this.onButtonClick)
```

Comment:
It does not work for me unless I add a second parameter **this** to register(), like this:
```javascript
this.events.register('buttonclick', this, this.onButtonClick)
```